biome.json

```diff
@@ -0,0 +1,97 @@
+{
+  "$schema": "https://biomejs.dev/schemas/1.8.3/schema.json",
+  "vcs": {
+    "clientKind": "git",
+    "enabled": true,
+    "useIgnoreFile": true
+  },
+  "files": {
+    "ignoreUnknown": true
+  },
+  "formatter": {
+    "enabled": true,
+    "formatWithErrors": false,
+    "indentStyle": "space",
+    "indentWidth": 2,
+    "lineEnding": "lf",
+    "lineWidth": 80,
+    "ignore": [
+      "**/package.json",
+      "**/dist/**",
+      "**/.changeset",
+      "**/CHANGELOG.md",
+      "**/pnpm-lock.yaml",
+      "**/pnpm-workspace.yaml"
+    ]
+  },
+  "organizeImports": { "enabled": true },
```

Comment:
the import sorter is quite interesting, it sorts import by the closet it you from the current file which is quite helpful once you get used to it.
Read more here https://biomejs.dev/analyzer/import-sorting/